pkg/kubelet/cm/cpumanager/fake_cpu_manager.go

```diff
@@ -60,6 +60,11 @@ func (m *fakeManager) GetTopologyHints(pod *v1.Pod, container *v1.Container) map
 	return map[string][]topologymanager.TopologyHint{}
 }
 
+func (m *fakeManager) GetPodTopologyHints(pod *v1.Pod) map[string][]topologymanager.TopologyHint {
+	klog.Infof("[fake cpumanager] Get Pod Topology Hints")
```

Comment:
Maybe reword the message in  `GetTopologyHints()` to explicitly say it is getting `Container Topology Hints`.

Reply:
Done